packages/template/typescript-webpack/src/TypeScriptWebpackTemplate.ts

```diff
@@ -34,13 +33,8 @@ class TypeScriptWebpackTemplate extends BaseTemplate {
           },
         },
       });
-
-      // Configure scripts for TS template
-      packageJSON.scripts.lint = 'eslint --ext .ts,.tsx .';
-
-      await fs.writeJson(packageJSONPath, packageJSON, { spaces: 2 });
+      await fs.writeFile(forgeConfigPath, `module.exports = ${JSON.stringify(forgeConfig, null, 2)}`);
```

Comment:
This makes the config Worse ™️ because it's exporting JSON which does nasty things like quote the keys.  Can we instead make this inject JS somehow (even just string replacement would be better IMO)

Reply:
[b1d85f8](https://github.com/electron/forge/pull/2991/commits/b1d85f8adb99311efa731b6fce479f3465cfd155)